js/index.js

```diff
@@ -120,5 +158,29 @@ $('.vowel-button').on('click', () => {
   }
 });
 
+$('.start-bonus-round').on('click', () => {
+  $('.popup-cover').css('display', 'none');
```

Comment:
lets make a dom.js function that does these css changes

Reply:
Should be all fixed now!